environs/bootstrap/bootstrap.go

```diff
@@ -513,10 +540,9 @@ func bootstrapImageMetadata(
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	// This constraint will search image metadata for all supported architectures and series.
 	imageConstraint := imagemetadata.NewImageConstraint(simplestreams.LookupParams{
 		CloudSpec: region,
-		Series:    availableTools.AllSeries(),
```

Comment:
we can't remove the series and arch constraints on image metadata look up or else we could an image which is incompatible with the tools